cfgov/unprocessed/css/video-player.less

```diff
@@ -141,7 +137,43 @@
         margin-top: @gutter-half__em;
     });
 
-    .respond-to-min(@bp-sm-min, {
+    .respond-to-range(@bp-sm-min, @fcm-bp-max {
         .grid_column(6);
     });
 }
+
+.respond-to-max(@fcm-bp-max, {
```

Comment:
I think so because the FCM less and breakpoints are contained in Capital Framework.